packages/audio-block/src/AudioBlock.tsx

```diff
@@ -0,0 +1,206 @@
+        if (doneAll && uploadResults) {
+            updateAudioAsset(uploadResults[0]);
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [doneAll, uploadResults]);
+
+    return (
+        <div data-test-id="audio-block" className={audioBlockClassNames}>
+            {audio ? (
+                <div className="tw-group">
+                    {isEditing && (
+                        <ItemToolbar
+                            onRemoveAsset={onRemoveAsset}
+                            onUploadClick={openFileDialog}
+                            onAssetChooseClick={openAssetChooser}
+                        />
+                    )}
+
+                    {isLoading ? (
+                        <div className="tw-flex tw-items-center tw-justify-center tw-h-14">
+                            <LoadingCircle />
+                        </div>
+                    ) : (
+                        <audio
+                            data-test-id="audio-block-audio-tag"
+                            key={audio.id}
+                            controls
+                            className={audioTagClassNames}
+                            controlsList="nodownload"
+                            preload="auto"
+                        >
+                            <source src={audio.genericUrl} type="audio/mp3" />
+                        </audio>
+                    )}
+                </div>
+            ) : (
+                isEditing && (
+                    <UploadPlaceholder
+                        onUploadClick={openFileDialog}
+                        onAssetChooseClick={openAssetChooser}
+                        loading={isLoading}
+                        setDroppedFiles={setDroppedFiles}
+                    />
+                )
+            )}
+            <div className="tw-flex tw-gap-4 tw-justify-between tw-w-full">
+                <div className="tw-flex-1">
+                    <RichTextEditor
+                        designTokens={designTokens ?? undefined}
+                        border={false}
+                        onBlur={saveTitle}
+                        placeholder={isEditing ? 'Asset name' : undefined}
+                        readonly={!isEditing}
+                        value={title ?? DEFAULT_CONTENT_TITLE}
+                        plugins={customTitlePlugins}
+                    />
+                    <RichTextEditor
+                        designTokens={designTokens ?? undefined}
+                        border={false}
+                        position={Position.FLOATING}
+                        onBlur={saveDescription}
+                        placeholder={isEditing ? 'Add a description here' : undefined}
+                        readonly={!isEditing}
+                        value={description ?? DEFAULT_CONTENT_DESCRIPTION}
+                    />
+                </div>
+                {audio && <BlockAttachments assetToDownload={audio} appBridge={appBridge} />}
```

Comment:
Careful with this kind of conditional, prefer ternary (`audio ? <... /> : null`) instead.
If we add another condition, `audio && ... && <... />` we might have a magic `0`/an error that appears.

Reply:
changed to ternary [commit](https://github.com/Frontify/guideline-blocks/pull/475/commits/44d111e0bb9690e57160c152ebb17efc3d094b7f)